pom.xml

```diff
@@ -372,20 +374,6 @@
                     <artifactId>maven-jxr-plugin</artifactId>
                     <version>2.5</version>
                 </plugin>
-                <plugin>
```

Comment:
This was the reason that PMD wasn't running.  I'm not sure why yet, but this plugin being declared twice (around line 212 and 375) caused pmd not to run.  There were warnings about the duplicate plugin at the top of the build output, but of course, they aren't accumulated and re-printed at the bottom of the build output, so it is easy to overlook.